pikaur/search_cli.py

```diff
@@ -138,10 +140,22 @@ def get_sort_key(pkg: AURPackageInfo) -> float:
                     package.popularity
                 ), 3)
 
+            color_config = PikaurConfig().colors
+            version_color: int = color_config.get('Version')  # type: ignore
+            version = package.version
+
+            if getattr(package, "outofdate", None) is not None:
+                version_color: int = color_config.get('VersionDiffOld')  # type: ignore
+                version = "{} [{}: {}]".format(
```

Comment:
just noticed what outofdate label looks a bit sticked:
![2018-04-12--1523566534_1235x709_scrot](https://user-images.githubusercontent.com/1655669/38703852-b3f540ee-3ea4-11e8-843f-a7f9f6041d15.png)

what about version having always the same color but `outfdate` label and date have `VersionDiffOld` color?

Reply:
They're related that's why I made them the same colour. But sure if you think that looks better we could do it that way. Would you want me to make another pr or do you just want to make a quick commit?

Reply:
as for me what's not that much important, was just saying

btw 2ds xl is quite cheap, could be a better experience than emulator :3

Reply:
Wrong place?

P.S. I have a 3ds already thanks :P

Reply:
i mentioned 2ds because emulator can't do 3d as well, sorry for the offtopic :D